src/topology/algebra/field.lean

```diff
@@ -0,0 +1,27 @@
+/-
+Copyright (c) 2021 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import topology.algebra.ring
+import topology.algebra.group_with_zero
+
+/-!
+We don't have a separate typeclass for topological fields:
+it's enough to have `{𝕜 : Type*} [field 𝕜] [topological_space 𝕜] [topological_ring 𝕜]`.
```

Comment:
it's not really true: we have `has_continuous_inv` which should also be satisfied by topological fields. But it is true that your lemma does not require it. Can you just clarify this sentence? (I agree that this file is a good home for it).